pandas/tests/test_strings.py

```diff
@@ -728,10 +728,6 @@ def test_count(self):
             ["foo", "foofoo", np.nan, "foooofooofommmfoo"], dtype=np.object_
         )
 
-        result = strings.str_count(values, "f[o]+")
```

Comment:
can you also create an issue to either make pandas/tests/strings/test_object.py or add a fixture to support th emultiple implementations (maybe better)